### PR TITLE
remove envelope

### PR DESCRIPTION
**Pull request by adecaro**
This PR does the following: It removes the need to register the envelope at the network interface level. This reduces further the dependencies with the underlying backend.

### CODE REVIEW

File: token/services/ttx/accept.go [hunk 1]
```diff
@@ -40,18 +40,6 @@ func (s *AcceptView) Call(context view.Context) (interface{}, error) {
...
 		return nil, errors.Wrap(err, "failed to marshal token request")
 	}
-	// Check the envelope exists
-	env := s.tx.Payload.Envelope
-	if env == nil {
-		return nil, errors.Errorf("expected fabric envelope")
-	}
-
-	// Store envelope
-	if !s.options.SkipApproval {
```
Comment (alexandrosfilios) [inline token/services/ttx/accept.go:50]: We can remove the field `SkipApproval`

File: token/services/ttx/transaction.go [hunk 2]
```diff
@@ -291,25 +290,13 @@ func (t *Transaction) TMSID() token.TMSID {
 }
 
 func (t *Transaction) setEnvelope(envelope *network.Envelope) error {
```
Comment (alexandrosfilios) [inline token/services/ttx/transaction.go:292]: We don't need this method I think

File: token/services/ttx/accept.go (hunk 1 above)
Comment (adecaro) [reply token/services/ttx/accept.go:50]: it is still used in other places.

**Review by alexandrosfilios — APPROVED** [review]
LGTM

File: token/services/ttx/transaction.go (hunk 2 above)
Comment (adecaro) [reply token/services/ttx/transaction.go:292]: 👍

**Pull request merged**